roles/openshift_grafana/templates/grafana-ocp.yaml

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
The `templates` dir is for jinja2 templates (`.j2` files). For a Kube template please place the file in the role's `flies/` directory